### ISSUE TITLE
Contact Us-Mobile: Icon's should be aligned vertically centered on devices.

### ISSUE THREAD

**Issue by mahender-sowedane** [titan-cebu/Tricon-Drilling-Solutions#80]
Devices:
Ipad (Portrait)
Itouch (Portrait & Landscape)
S4 (Portrait & Landscape)

![screenshot_2015-11-17-09-46-03](https://cloud.githubusercontent.com/assets/13309730/11205135/43ad4962-8d2b-11e5-9bb2-ea7f3c25a6d0.png)

**Comment by mahender-sowedane** [comment]
Still not fixed

![2015-11-20_1542-res702](https://cloud.githubusercontent.com/assets/13309730/11298338/2f1c2ede-8fa4-11e5-93ac-c8686777db7f.png)

**Comment by nakul-titan** [comment]
Device name please.

**Comment by mahender-sowedane** [comment]
Yes, by decreasing the widths of the browsers icon's should be remains same
thanks!

**Comment by mahender-sowedane** [comment]
Still not fixed

![bug01](https://cloud.githubusercontent.com/assets/13309730/11358363/f663e55a-9296-11e5-956c-516aa8abc678.png)

**Comment by roseliesromeo** [comment]
![image](https://cloud.githubusercontent.com/assets/11186313/11492345/d57a8b92-9826-11e5-83a1-ad2179b9697a.png)
